src/main/webapp/data/userMapData.json

```diff
@@ -687,7 +685,6 @@
       "Universidad Nacional de San Agustín"
     ],
     "Philippines": [
-      "aGreatlife",
```

Comment:
Checked on the web it doesn't exist.

Reply:
"aGreatlife" seems to register a hit for "aGreatLife Online Store" in the Philippines. Although this is not an educational institute, we can include this in our user map (much like the one in https://github.com/TEAMMATES/teammates/pull/6605#discussion_r100798901) if it is a valid use case. It may still be best to direct this to @damithc for confirmation however.

Reply:
Yes, we can keep that.